vmp/resource_customer_user.go

```diff
@@ -39,12 +39,9 @@ func resourceCustomerUser() *schema.Resource {
 				Optional: true,
 				Default:  false,
 				DiffSuppressFunc: func(k, old, new string, d *schema.ResourceData) bool {
```

Comment:
If DiffSuppressFunc returns true, then Terraform ignores the change in the field. This function makes IsAdmin a "write-only" field because that cannot be changed once a user is created. 
If len(old) >0 is true it means that the user is exists so we suppress differences in IsAdmin
If len(old) = 0 then it means the user is new (a new resource) so we do not suppress the diff.

Reply:
got it ty for the explanation.